services/src/main/java/org/exoplatform/webconferencing/externalvisio/dao/ExternalVisioConnectorDAO.java

```diff
@@ -50,4 +50,15 @@ public List<ExternalVisioConnectorEntity> getActiveExternalVisioConnectorsForSpa
     }
 
   }
+
+  public List<ExternalVisioConnectorEntity> getActiveExternalVisioConnectorsForUser() {
+    TypedQuery<ExternalVisioConnectorEntity> query =
+                                                   getEntityManager().createNamedQuery("ExternalVisioConnector.getActiveExternalVisioConnectorsForUsers",
+                                                                                       ExternalVisioConnectorEntity.class);
+    try {
+      return query.getResultList();
+    } catch (NoResultException e) {
+      return Collections.emptyList();
+    }
+  }
```

Comment:
can we merge this function with the previous one : 
public List<ExternalVisioConnectorEntity> getActiveExternalVisioConnectors(boolean forSpaces) {
   String queryName = "ExternalVisioConnector.getActiveExternalVisioConnectorsFor"
   if (!forSpaces) {
      queryName+="Users";
   } else {
      queryName+="Spaces";
   }
   TypedQuery<ExternalVisioConnectorEntity> query =
                                                   getEntityManager().createNamedQuery(queryName,
                                                                                       ExternalVisioConnectorEntity.class);

....
}